manager/src/main/resources/application.properties

```diff
@@ -95,3 +95,4 @@ managed-connectors.auth.offline-token=${MC_AUTH_OFFLINE_TOKEN:fakeOfflineToken}
 # RHOAS
 rhoas.timeout-seconds=60
 rhoas.ops-account.client-id=fakeClientId
+rhoas.topic-prefix=ob-
```

Comment:
This is the default value so it should not be necessary

Reply:
Refactored this part